src/controller/java/src/chip/devicecontroller/ControllerParams.java

```diff
@@ -0,0 +1,27 @@
+package chip.devicecontroller;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+/** Parameters representing initialization arguments for {@link ChipDeviceController}. */
+public final class ControllerParams {
+  private final int udpListenPort;
+
+  /** Creates a {@link ControllerParams} with default values. */
+  public ControllerParams() {
+    this(5543 + 1 /* To match old behavior of CHIP_PORT + 1 */);
```

Comment:
should we name this constant, like `LEGACY_CHIP_PORT` ?

I am also surprised about 5543 ... it used to be 5540.

Reply:
We seem to have:

```
src/lib/core/CHIPConfig.h -> #define CHIP_PORT 5540
examples/placeholder/linux/apps/app1/include/CHIPProjectConfig.h -> #define CHIP_PORT 5543
examples/tv-app/tv-common/include/CHIPProjectAppConfig.h -> #define CHIP_PORT 5640
```

I believe the core one is the "typical legacy" so lets use 5540 as the legacy default here and add the +1 to preserve previous behavior

Reply:
Looks like you already made the change, thank you!